x-pack/plugins/ml/common/util/runtime_field_utils.test.ts

```diff
@@ -7,96 +7,65 @@
 
 import { isRuntimeField, isRuntimeMappings } from './runtime_field_utils';
 
-describe('ML runtime field utils', () => {
```

Comment:
I suspect something went wrong with a master rebase/merge. The removed lines are actually the newer code merged as part of this PR: https://github.com/elastic/kibana/pull/95651/files#diff-02075861bae59012670e1e16f2017e22cfca82cf3d7a210eb9c66e96503252d1R10

Reply:
Updated here [`dbd5e91` (#95623)](https://github.com/elastic/kibana/pull/95623/commits/dbd5e9121f681bf8ef749824de71aa396355e38f)